browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -28,7 +28,10 @@ class MarkdownNoteDetail extends React.Component {
         content: ''
       }, props.note),
       isLockButtonShown: false,
-      isLocked: false
+      isLocked: false,
+      fullScreen: false,
+      widthOfNoteDetail: 0,
+      widthOfMainBody: 0
```

Comment:
Yeah I was concerned about that from the start. I ended up moving the logic/event into `browser/main/Main.js` like you suggested. I just have emit's inside the `MarkdownNoteDetail` and the snippet file. I tried to follow the code from the PR you recommend.

Also, thanks for all the help!